NEWS.md

```diff
@@ -1,3 +1,6 @@
+# iglu 4.3.0
+* Updated MAG computation in the presence of large missing data
```

Comment:
add that the default for MAG has also been changed to do differences over meter frequency (previous default was hour) or something like that